sanic_openapi/spec.py

```diff
@@ -1,47 +1,89 @@
-from sanic import Sanic
+from typing import Dict
 
 
 class Spec:
-    def __init__(self, app: Sanic) -> None:
+    def __init__(self, config) -> None:
+        self.config = config
         self.swagger = "2.0"
+        self._tags = {}
+        self._paths = {}
+        self._definitions = {}
```

Comment:
Looks duplicated with line 62-64. I think using `self.tags`, `self.paths`, and `self.definitions` are better.

Reply:
Ah, I see the `@property` methods, please ignore this comment. 😝